### ISSUE TITLE
szilard/benchm-ml (33015554)

### ISSUE THREAD

**Issue by gfleetwood** [gfleetwood/asteres#1755]
https://github.com/szilard/benchm-ml

A minimal benchmark for scalability, speed and accuracy of commonly used open source implementations (R packages, Python scikit-learn, H2O, xgboost, Spark MLlib etc.) of the top machine learning algorithms for binary classification (random forests, gradient boosted trees, deep neural networks etc.).
